### PR TITLE
Reslve conversion

### PR DESCRIPTION
**Pull request by happyveeru**
Resolve conversation pull request

### CODE REVIEW

File: VeeraGitTest/src/com/veera/test/M4.java [hunk 1]
```diff
@@ -4,7 +4,7 @@ public class M4 {
 
 	public static void main(String[] args) {
 		// TODO Auto-generated method stub
-
+		System.out.println("REsove conversation commit");
```
Comment (happyveeru) [inline VeeraGitTest/src/com/veera/test/M4.java]: Is this needed

**Review by happyveeru — COMMENTED** [review]
Please make suggested changes

File: VeeraGitTest/src/com/veera/test/M4.java (hunk 1 above)
Comment (happyveeru) [reply VeeraGitTest/src/com/veera/test/M4.java]: @valid not needed here, removed and code committed

Comment (happyveeru) [reply VeeraGitTest/src/com/veera/test/M4.java]: @valid not needed, removed and committed

Comment (happyveeru) [reply VeeraGitTest/src/com/veera/test/M4.java]: valid annotation not needed here, removed and code committed

Comment (happyveeru) [reply VeeraGitTest/src/com/veera/test/M4.java]: Hey this is second reply .. not needed and committed

File: VeeraGitTest/src/com/veera/test/M4.java [hunk 2]
```diff
@@ -4,7 +4,12 @@ public class M4 {
 
 	public static void main(String[] args) {
 		// TODO Auto-generated method stub
-
+		// System.out.println("REsove conversation commit");
```
Comment (2veeru) [inline VeeraGitTest/src/com/veera/test/M4.java:7]: Start Second Review

File: VeeraGitTest/src/com/veera/test/M4.java [hunk 3]
```diff
@@ -4,7 +4,12 @@ public class M4 {
...
+		// Additional
```
Comment (2veeru) [inline VeeraGitTest/src/com/veera/test/M4.java]: Comment can be removed

**Review by 2veeru — CHANGES_REQUESTED** [review]
Start Second Review

File: VeeraGitTest/src/com/veera/test/M4.java (hunk 3 above)
Comment (happyveeru) [reply VeeraGitTest/src/com/veera/test/M4.java]: Comment removed

File: VeeraGitTest/src/com/veera/test/M4.java (hunk 2 above)
Comment (happyveeru) [reply VeeraGitTest/src/com/veera/test/M4.java:7]: Made second review comments

File: VeeraGitTest/src/com/veera/test/M4.java (hunk 3 above)
Comment (2veeru) [reply VeeraGitTest/src/com/veera/test/M4.java]: Comments not properly removed

File: VeeraGitTest/src/com/veera/test/M4.java [hunk 4]
```diff
@@ -4,7 +4,11 @@ public class M4 {
 
 	public static void main(String[] args) {
 		// TODO Auto-generated method stub
-
+		// System.out.println("REsove conversation commit");
+		// System.out.println("REsove conversation commit");
+				// Additional
+		// System.out.println("REsove conversation commit");
+				// Additional
```
Comment (2veeru) [inline VeeraGitTest/src/com/veera/test/M4.java:11]: here comment still pending

**Review by 2veeru — CHANGES_REQUESTED** [review]

File: VeeraGitTest/src/com/veera/test/M4.java (hunk 4 above)
Comment (happyveeru) [reply VeeraGitTest/src/com/veera/test/M4.java:11]: I already did comment

File: VeeraGitTest/src/com/veera/test/M4.java (hunk 3 above)
Comment (happyveeru) [reply VeeraGitTest/src/com/veera/test/M4.java]: Already removed

**Comment by happyveeru** [comment]
Capture2

**Pull request merged**